### PR TITLE
Import metrics_center

### PR DESCRIPTION
**Pull request by dnfield**
Related: https://github.com/flutter/flutter/pull/74535

Only changes between this and the one in the flutter repo is an updated pubspec version number and URL, added a changelog as well.

We should further:

- Make sure all public members are documented
- Add example(s)

### CODE REVIEW

**Review by liyuqian — APPROVED** [review]
LGTM. Thanks for the move! I'll add more docs and examples once the metrics_center migration is finalized.

**Comment by dnfield** [comment]
Analysis won't pass without docs. I'm going to try to add some.

File: packages/metrics_center/lib/src/constants.dart [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Strings used for GIT tags
```
Comment (liyuqian) [inline packages/metrics_center/lib/src/constants.dart:5]: It's probably more accurate to have `GIT tags` -> `[MetricPoint] tags`

**Review by liyuqian — COMMENTED** [review]
Thanks for adding the docs and fixing the analyze issues!

File: packages/metrics_center/lib/src/constants.dart (hunk 1 above)
Comment (dnfield) [reply packages/metrics_center/lib/src/constants.dart]: Done.

A lot of these docs are ... sparse. But it's better than nothing at all I guess.

**Pull request merged**